src/Traits/Categorizable.php

```diff
@@ -40,18 +42,10 @@ public static function bootCategorizable(): void
     public function initializeCategorizable(): void
     {
         $unguarded = array_diff(
-            Schema::getColumnListing($this->getTable()),
+            static::$columnListing ?? static::$columnListing = Schema::getColumnListing($this->getTable()),
             $this->getGuarded()
         );
 
-        if (in_array(HasAdditionalColumns::class, class_uses_recursive($this))) {
-            $unguarded = array_merge($unguarded, $this->getAdditionalColumns()->pluck('name')->toArray());
```

Comment:
if you remove this, the additional columns aren't fillable anymore, therefore tests fail supposedly.